test/powershell/Modules/Microsoft.PowerShell.Utility/FormatHex.Tests.ps1

```diff
@@ -14,115 +14,127 @@
         Hexadecimal equivalent of the input data is displayed.
 #>
 
-function RunInputObjectTestCase($testCase)
+function RunInputObjectTestCase($testCases)
```

Comment:
I think that these functions should just be inlined into the appropriate context block. Having these functions only makes things a little harder to find. it allows the test case data be closer to the actual test.